dataplane/sdx/sdx.go

```diff
@@ -186,11 +187,11 @@ func DeleteSdx(c *cli.Context) {
 	if err != nil {
 		utils.LogErrorAndExit(err)
 	}
-	log.Infof("[deleteSdx] SDX cluster deleted in environment: %s", name)
+	log.Infof("[DeleteSdx] SDX cluster deleted in environment: %s", name)
 }
 
 func RepairSdx(c *cli.Context) {
-	defer commonutils.TimeTrack(time.Now(), "delete SDX cluster")
+	defer commonutils.TimeTrack(time.Now(), "Sdx cluster repair")
```

Comment:
Lets keep it consistent, if SDX is all caps in other messages, it should be also all caps here.

Reply:
sure, I'll need to update other text's as well.